src/mcp_atlassian/utils/logging.py

```diff
@@ -6,9 +6,13 @@
 """
 
 import logging
+import sys
+from typing import TextIO
 
 
-def setup_logging(level: int = logging.WARNING) -> logging.Logger:
+def setup_logging(
+    level: int = logging.WARNING, stream: TextIO = sys.stderr
+) -> logging.Logger:
```

Comment:
Consider adding the new parameter to the docstring below to document the stream parameter and its purpose.